### DISCUSSION TITLE
开启和关闭审计日志 - DaoCloud Enterprise

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1003] [Announcements]
# 开启和关闭审计日志 - DaoCloud Enterprise

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/ghippo/user-guide/open-audit/

**Comment by liyuerich** [comment]
配置 API 服务器 
其中的添加命令能否有具体的解释，比如 30， 1， 100的具体含义

**Reply by yitingdc** [reply]
用 https://gitlab.daocloud.cn/ndx/ghippo/-/issues/634 跟踪， 整个开启关闭方法这个月准备理一下

**Reply by yitingdc** [reply]
关于开启/关闭采集 k8s 审计日志，该文档已更新https://docs.daocloud.io/ghippo/user-guide/audit/open-audit/
当前还剩一个遗留任务(优化 https://docs.daocloud.io/ghippo/user-guide/audit/open-k8s-audit/ )，
等待insight同事去更新：
https://gitlab.daocloud.cn/ndx/engineering/insight/insight/-/issues/1964

**Reply by liyuerich** [reply]
我已经晕了， 文档站地址不是 https://docs.daocloud.io/ghippo/user-guide/audit/open-k8s-audit/#_1 吗？

**Reply by yitingdc** [reply]
url 已经更新成对的了

**Comment by gssjl2008** [comment]
开启前后、系统组件资源使用率会有多大提升，可以完善下
